### DISCUSSION TITLE
6.3.0 without GNU autotools support. Why?🤔

### DISCUSSION THREAD

**Discussion by kloczek** [johnkerl/miller#1053] [Q&A] [answered]
Is it any reason why GNU autotools build framework support have been removed? 🤔
Looks like like huge step back ,,

**Comment by johnkerl** [comment]
@kloczek have you used 6.0.0 or 6.1.0 or 6.2.0? autotools (as I understand the word) was part of the C implementation(Miller 5 and below); it's not part of the package in the Go port (Miller 6 and above) ...

**Comment by kloczek** [comment]
Yes I have and it was workimg perfectly fine.
I'm taking about GNU automake/autoconf support.

**Comment by johnkerl** [comment]
@kloczek we can't be talking about the same thing.

The C code and associated automake files were all removed here -- before the 6.0.0 release:
https://github.com/johnkerl/miller/pull/731/files

To help me understand, can you show me an example of what "working perfectly fine" meant for you in, say, 6.2.0, and what a `git log` and/or `ls -l` looks like in that situation? Can you show me how you used autotools in 6.2.0 -- what commands you used, what output you got?

**Comment by johnkerl** [comment, marked as answer]
Marking as answered pending a response

The only plausible explanation is a stale checkout

Miller 6 and above doesn't have autotools since it is no longer written in C. It is now written in Go, and uses the Go build system.
